pkg/lib/index/catalog.go

```diff
@@ -0,0 +1,53 @@
+package indexer
```

Comment:
nit: although there are some exceptions, this probably shouldn't be in the `lib` folder since it contains OLM specific logic.